### PR TITLE
Worker logs: sampleTimeout job

### PR DESCRIPTION
**Pull request by pallavi2209**

### CODE REVIEW

**Comment by pallavi2209** [comment]
@iamigo Yes this is for sampleTimeout job, worker logging. But I assumed we are covering clock logging as a part of worker logging, because clock activities are executed in worker dynos. I didn't know there is different story for clock logging as well.
Let me update the PR title to avoid confusion.

File: worker/jobProcessor.js [hunk 1]
```diff
@@ -76,9 +76,34 @@ jobQueue.process(jobType.BULKUPSERTSAMPLES, (job, done) => {
...
+      objToReturn.recordCount = dbRes.numberTimedOut;
+
+      // errorCount = number of samples that did not time out
+      objToReturn.errorCount = dbRes.numberEvaluated - dbRes.numberTimedOut;
+      objToReturn.workTime = dbEndTime - jobStartTime;
+      objToReturn.reqStartTime = reqStartTime;
+      done(null, objToReturn);
+    } else {
+      done();
+    }
```
Comment (shriramshankar) [inline worker/jobProcessor.js]: Lines 94 to 103 seems similar to lines 53 to 69, expect for a few lines. Do you think moving them to the activityLogUtils will help?

Comment (pallavi2209) [inline worker/jobProcessor.js]: For current calculations of dbTime, queueTime and workTime, looks like we can use common method, but recordCount and errorCount can vary from job to job. Let's move them actually, will improve readability.

**Comment by pallavi2209** [comment]
@shriramshankar updated the PR

**Pull request merged**